examples/va/hci/kustomization.yaml

```diff
@@ -7,5 +7,54 @@ components:
   ## It's possible to replace ../../../va/hci/ with a git checkout URL as per:
   ## https://github.com/kubernetes-sigs/kustomize/blob/master/examples/remoteBuild.md
 
+# Add StorageMgmt network template, as it is needed for CephHCI
+patchesJson6902:
```

Comment:
FYI, though maybe you don't get the deprecation warning with 5.0?

```
[fultonj@jove hci{va2_sriov}]$ kustomize build . > ~/va2.yaml
# Warning: 'patchesJson6902' is deprecated. Please use 'patches' instead. Run 'kustomize edit fix' to update your Kustomization automatically.
[fultonj@jove hci{va2_sriov}]$ 

[fultonj@jove hci{va2_sriov}]$ kustomize version
v5.1.0
[fultonj@jove hci{va2_sriov}]$ 
```

Reply:
I will see what I can do to eliminate that

Reply:
Fixed